src/components/FormElements/XdrLedgerKeyPicker.tsx

```diff
@@ -526,25 +544,38 @@ export const MultiLedgerEntriesPicker = ({
   id,
   onChange,
   value = [""],
-  buttonLabel = "Add another ledger key",
+  allowMultiple,
   limit = 100,
+  activeTab = "ledgerKey",
+  readOnlyLedgerKey,
 }: MultiPickerProps) => {
-  const [activeTabId, setActiveTabId] = useState<string>("ledgerKey");
+  const [activeTabId, setActiveTabId] =
+    useState<LedgerKeyPickerType>(activeTab);
   if (!value || !value.length) {
     value = [];
   }
 
+  const defaultLedgerKeyField =
+    readOnlyLedgerKey && getLedgerKeyFields(readOnlyLedgerKey);
+
+  const tabs = [
+    {
+      id: "ledgerKey",
+      label: defaultLedgerKeyField
+        ? `Use ${defaultLedgerKeyField?.label} Key`
+        : "Select Ledger Key",
```

Comment:
Nit: to make sure the label will be there.

```suggestion
      label: defaultLedgerKeyField?.label
        ? `Use ${defaultLedgerKeyField.label} Key`
        : "Select Ledger Key",
```